pb/proof.proto

```diff
@@ -0,0 +1,11 @@
+syntax="proto3";
+
+package proof.pb;
+
+message Proof {
+  int64 start     = 1;
+  int64 end       = 2;
+  repeated bytes nodes = 3;
+  bytes leafHash = 4;
+  bool isMaxNamespaceIDIgnored=5;
+}
```

Comment:
[optional] doc comments on what each field does would be super helpful. Content from the specs may help https://github.com/celestiaorg/nmt/blob/master/docs/spec/nmt.md#namespace-proof

Reply:
Hey @rootulp, thanks for the great suggestion. Added comments here https://github.com/celestiaorg/nmt/pull/220/commits/223b55abd6d47090ac31b5d18746950db1649d08